.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @david31982 @holgiosalos @JuanDelahozMiranda @manuelq12 @valeryibarra @cposada23
```

Comment:
Add a blank line to the end of the file to remove this warning:
![image](https://user-images.githubusercontent.com/10209180/168046961-7dd5026a-b0ed-42be-b415-7972de5f139b.png)

You can configure visual studio code to do this automatically:
https://stackoverflow.com/a/44704969

Reply:
Done